docker/php/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:7.3-cli
```

Comment:
Could you use the images from https://github.com/yiisoft/yii-docker - I think there's no need for a `Dockerfile` atm.

Reply:
I thought about it, but these would set up so much stuff not needed for every repository (imagemagick, nodejs, ..)

Reply:
These should be the requirements for core modules (`requirements.php`), were there changes in Yii3 for this?

I know that node is debatable but in the long run I think it'll become a requirement like composer (at least for web apps).

Having the stuff ready in an image is one of the main purposes of Docker, IMO.

Reply:
All righty. I was aiming at a minimalistic setup for each repo requirements.
I'll update the PR to use the yii2 image.

Reply:
Done